docs/how-to-guides/advanced-user-guides/enable-user-retention.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Enabling User Retention
+---
+
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/how-to-guides/advanced-user-guides/enable-user-retention.md"/>
+</head>
+
+In Rancher v2.8.5 and later (and v2.7.14 and later in the v2.7.x line), you can enable user retention to automatically disable or delete inactive user accounts after a configurable time period.
```

Comment:
```suggestion
In Rancher v2.8.5 and later, you can enable user retention to automatically disable or delete inactive user accounts after a configurable time period.
```

We should minimize cross-version references since users shouldn't reading version X docs aren't going to be concerned about version Y (especially if it's older).